.github/workflows/build-deploy-dev.yml

```diff
@@ -0,0 +1,48 @@
+on:
+  push:
+    branches:
+      - master
+jobs:
+  deploy_build_dev:
+    name: Build container and deploy
+    env:
+      AZURE_CONTAINER_REGISTRY_SERVER: ${{ secrets.AZURE_CONTAINER_REGISTRY_SERVER }}
+    runs-on: ubuntu-latest
+    needs: test
+    if: github.ref == 'refs/heads/master'
```

Comment:
might not need this if anymore? (I'm not sure)

Reply:
yeah I'll remove them